### PR TITLE
Upgrades to vault 0.6.4

### PR DESCRIPTION
**Pull request by JosephSalisbury**

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -1,9 +1,11 @@
 package: github.com/giantswarm/cert-operator
 import:
 - package: github.com/golang/protobuf
-  version: 8ee79997227bf9b34611aee7946ae64735e6fd93
+  version: 2bba0603135d7d7f5cb73b2125beeda19c09f4ef
+- package: google.golang.org/grpc
```
Comment (JosephSalisbury) [inline glide.yaml]: This is one of those dumb vanity go library things, and seemed to screw glide up

File: glide.yaml [hunk 2]
```diff
@@ -1,9 +1,11 @@
 package: github.com/giantswarm/cert-operator
 import:
 - package: github.com/golang/protobuf
-  version: 8ee79997227bf9b34611aee7946ae64735e6fd93
+  version: 2bba0603135d7d7f5cb73b2125beeda19c09f4ef
```
Comment (JosephSalisbury) [inline glide.yaml]: There was a library conflict here too

File: glide.yaml [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
+- package: google.golang.org/grpc
+  version: b5071124392bfd416c713e6595ac149b387f7186
 - package: github.com/giantswarm/certctl
-  version: 350bd8e6c1098d0c20166ef7e286c8c7afe6e517
+  version: 0b15172fab3ef7d0805680a90876a7d704e2e80e
```
Comment (JosephSalisbury) [inline glide.yaml]: This is our latest certctl version, which support 0.6.4

File: glide.yaml (hunk 1 above)
Comment (rossf7) [inline glide.yaml]: Ugh thats frustrating. Good spot!

**Pull request merged**